src/app/views/editors/ckeditor/ckeditor_preparer.ts

```diff
@@ -47,8 +47,6 @@ class CkeditorPreparer implements EditorPreparer {
     // TODO: ACF: http://sdk.ckeditor.com/samples/acf.html
     /*allowedContent: { 'b i li ul ol table thead tbody tr': true, 'h1 h2 h3 h4 p th td': {  styles: 'text-align,text-decoration' },
      a: {attributes: '!href,target'} img: { attributes: '!src,alt', styles: 'width,height', classes: 'left,right' } },*/
-    // TODO: enable images
-    // TODO: add ImageButton to removeButtons list
```

Comment:
agree, that those two are now fixed? or what was exactly meant by "enable images"?